jetty-ee10/pom.xml

```diff
@@ -50,22 +50,22 @@
     <jakarta.mail.api.version>2.1.3</jakarta.mail.api.version>
     <jakarta.servlet.api.version>6.0.0</jakarta.servlet.api.version>
     <jakarta.servlet.jsp.api.version>3.1.1</jakarta.servlet.jsp.api.version>
-    <jakarta.servlet.jsp.jstl.api.version>3.0.0</jakarta.servlet.jsp.jstl.api.version>
+    <jakarta.servlet.jsp.jstl.api.version>3.0.2</jakarta.servlet.jsp.jstl.api.version>
```

Comment:
The 3.0.2 version release of jstl was to match servlet 6.1, ee11. So don't upgrade that one for ee10.

Reply:
done with 1c513636e8f